modules/meeting/config/locales/en.yml

```
@@ -255,6 +256,7 @@ en:
       working_days: "Every working day"
       weekly: "Weekly"
     end_after:
+      never: "never"
       specific_date: "a specific date"
       iterations: "a number of occurrences"
```

Paired with changing line 68 to: "End series" instead of "End series after"

```suggestion
      never: "never"
      specific_date: "after a specific date"
      iterations: "after a number of occurrences"
```